Tools-Override/tests.targets

```diff
@@ -199,15 +199,20 @@
 
     <PropertyGroup Condition="'$(UseDotNetNativeToolchain)' == 'true'">
       <ILCBuildType Condition="'$(ILCBuildType)' == ''">ret</ILCBuildType>
+      <_TestILCFolder>$(TestILCFolder)</_TestILCFolder>
+      <!--When archiving tests, ilc.exe will be located at %RUNTIME_PATH%\..\TestILC-->
+      <_TestILCFolder Condition="'$(ArchiveTests)' == 'true'">%RUNTIME_PATH%\..\TestILC</_TestILCFolder>
+      <!--Temporarily comment out the test command line when executing on Helix so that we can analyze how many projects fail ilc compilation-->
+      <TestCommandLine Condition="'$(ArchiveTests)' == 'true'">:: $(TestCommandLine)</TestCommandLine>
```

Comment:
For testing did you want to change this to "echo" so we can more easily see what it would have done in the logs?